components/chart/accessibility/wai-aria-support.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Wai-Aria Support
+page_title: Telerik UI for Blazor Chart Documentation | Chart  Accessibility
+description: "Get started with the Telerik UI for Blazor Chart and learn about its accessibility support for WAI-ARIA, Section 508, and WCAG 2.1."
+tags: telerik,blazor,accessibility,wai-aria,wcag
+slug: chart-wai-aria-support 
+position: 50 
+---
+
+# Blazor Chart Accessibility
+
+@[template](/_contentTemplates/common/parameters-table-styles.md#table-layout)
+
+
+
+Out of the box, the Telerik UI for Blazor Chart provides extensive accessibility support and enables users with disabilities to acquire complete control over its features.
+
+
+The Chart is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.2  AA](https://www.w3.org/TR/WCAG22/) standards](https://www.w3.org/TR/WCAG22/) and [Section 508](http://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
```

Comment:
Again, a weird link issue with the template used for generating the page. Note that it affects multiple live pages.
```suggestion
The Chart is compliant with the [Web Content Accessibility Guidelines (WCAG) 2.2 AA](https://www.w3.org/TR/WCAG22/) standards and [Section 508](http://www.section508.gov/) requirements, follows the [Web Accessibility Initiative - Accessible Rich Internet Applications (WAI-ARIA)](https://www.w3.org/WAI/ARIA/apg/) best practices for implementing the [keyboard navigation](#keyboard-navigation) for its `component` role, provides options for managing its focus and is tested against the most popular screen readers.
```